setup.py

```diff
@@ -45,6 +45,7 @@ def rel(*parts):
     'Framework :: Django :: 3.2',
     'Framework :: Django :: 4.0',
     'Framework :: Django :: 4.1',
+    'Framework :: Django :: 4.2',
```

Comment:
That's blocked by https://github.com/pypa/trove-classifiers/pull/128

Reply:
Will this break the build?

Reply:
I _think_ it'll prevent any releases

Reply:
OK, I'll wait for the fix in upstream than. Thanks for informing.

Reply:
No longer blocked now 🎉